draft/2022-03-09-this-week-in-rust.md

```diff
@@ -16,21 +16,42 @@ If you find any errors in this week's issue, [please submit a PR](https://github
 
 ### Official
 
+* [Rust Foundation](https://foundation.rust-lang.org/posts/2022-03-08-member-spotlight-1password/)
+* [Security advisory for the regex crate (CVE-2022-24713)](https://blog.rust-lang.org/2022/03/08/cve-2022-24713.html)
+
 ### Project/Tooling Updates
+
 * [Fornjot (Code-CAD in Rust) - Weekly Dev Log - 2022-W09](https://www.fornjot.app/blog/weekly-dev-log/2022-w09/)
 * [This week in Fluvio #24: the programmable streaming platform](https://www.fluvio.io/news/this-week-in-fluvio-0024/)
+* [This Month in Rust OSDev: February 2022 | Rust OSDev](https://rust-osdev.com/this-month/2022-02/)
+* [IntelliJ Rust Changelog #166](https://intellij-rust.github.io/2022/03/09/changelog-166.html)
+* [Rust-Analyzer Changelog #119](https://rust-analyzer.github.io/thisweek/2022/03/07/changelog-119.html)
+* [DE] [WebAssembly-Runtime: Wasmer 2.2 liefert mehr Leistung im Singlepass Compiler](https://www.heise.de/news/WebAssembly-Runtime-Wasmer-2-2-liefert-mehr-Leistung-im-Singlepass-Compiler-6533165.html)
 
 ### Research
 
+* [Safe Pin projections through view types](https://blog.yoshuawuyts.com/safe-pin-projections-through-view-types/)
+
```

Comment:
Perhaps this should go in Observations/Thoughts?

Reply:
Yoshua Wuyts' blog posts show up pretty regularly here, and I don't think we've ever filed them under research before.